addin-fixed-weights/program/src/instruction.rs

```diff
@@ -23,12 +24,14 @@ pub enum VoterWeightAddinInstruction {
     SetupVoterWeightRecord { },
     /// Sets up VoterWeightRecord owned by the program
     ///
-    /// 0. `[]` Realm account
-    /// 1. `[]` Governing Token mint
-    /// 2. `[]` Governing token owner
-    /// 3. `[writable]` VoterWeightRecord
-    /// 4. `[signer]` Payer
-    SetPartialVoting {
+    /// 0. `[]` The Governance program account
+    /// 1. `[]` Realm account
+    /// 2. `[]` Governing Token mint
+    /// 3. `[]` Governing token owner
+    /// 4. `[signer]` Authority account
+    /// 5. `[]` Governing Owner Record. PDA seeds (governance program): ['governance', realm, token_mint, token_owner]
+    /// 6. `[writable]` VoterWeightRecord
+    SetVoterPercentage {
```

Comment:
The `SetVotePercentage` instruction in addin-fixed-weight and addin-vesting should be equivalent. Otherwise, we should form different instructions in UI depending on the active addin.
Equivalenting include next points:
- same list of accounts
- same parameters
- identical serialized data (in this case we need to serialize enum value in the same values)
- it will be good if the names will be the same (SetVote**r**Percentage)

To make this instruction equivalent we can specify `mint` value instead of `The Vesting account`. Moreover, we need to do this due to conceptual reason - SetVotePercentage instruction works with VoterWeightRecord, not VestingRecord. For one VoterWeightRecord can be exists different VestingRecord.

So, I suggest:
1. rename and reorganize the list of account for this instruction according to addin-vesting::SetVotePercentage
2. change VoterRecord to mint in addin-vesting::SetVotePercentage
3. investigate how we can serialize SetVotePercentage with identical enum value. If it hard to do, make some dummy values for addin-fixed-weight::instructions enum.